docs/how-to-work-on-the-docs-theme.md

```diff
@@ -7,19 +7,19 @@
 
 ## About the documentation site
 
-While you do not need to generate the documenatation site locally, sometimes it may be required for instance if you are working on the site theme or a feature for the site.
+While you do not need to generate the documentation site locally, sometimes it may be required for instance if you are working on the site theme or a feature for the site.
 
 The site is generated using [`docsify`](https://docsify.js.org), and served using GitHub pages.
 
-Typically you would not need to change any configuration, or build the site locally, but incase you are interested here is how it works:
+Typically you would not need to change any configuration or build the site locally, but incase you are interested here is how it works:
```

Comment:
Since we are changing this, let's make the wording a bit more clear.
```suggestion
Typically you would not need to change any configuration or build the site locally.  In case you are interested, here is how it works:
```

Reply:
Great. Looks like there is an extra space before "In case".

Reply:
@Ashikpaul I removed the space, so you should be able to commit the suggested change now.